profiles/arch/arm/armv5te/use.mask

```diff
@@ -1,2 +1,10 @@
 # Unmask armv5 specific USE flags
 -armv5te
+
+-cpu_flags_arm_v4
+-cpu_flags_arm_v5
+
+-cpu_flags_arm_edsp
+-cpu_flags_arm_thumb
+-cpu_flags_arm_vfp
+-cpu_flags_arm_vfp-d16
```

Comment:
Since we don't support anything pre armv5, I'd assume vfp means vfpv2

vfp-d16 seems to be for 16 double registers vfpv3 on armv7, so should be masked here I think